std/exception.d

```diff
@@ -2145,3 +2145,58 @@ pure nothrow @safe unittest
 
     auto infSlice = infinite[0 .. $]; // this would throw otherwise
 }
+
+
+/**
+ * Convenience mixin for trivially sub-classing exceptions
+ *
+ * Even trivially sub-classing an exception involves writing boilerplate code
+ * for the constructor to correctly pass in the source file and line number the
+ * exception was thrown from. This mixin makes it easier.
+ *
+ * Note that if you want to pass in special arguments to your exception, you
+ * obviously can't use this mixin and have to write the constructor manually.
+ */
```

Comment:
I experimented with the ddoc generation for this a bit, and it looks like the constructors' documentation doesn't end up in the exception's documentation unless the mixin statement has a ddoc comment on it. So, I think that you should mention that in the documentation and possibly put `///` on the mixin statement in your example.